unit_tests/test_lib_orbital_maneuvers.ks

```diff
@@ -0,0 +1,173 @@
+@lazyglobal off.
```

Comment:
Needs // This file is distributed under the terms of the MIT license, (c) the KSLib team